.github/workflows/ansible-test-sanity.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: Sanity
+on:
+  pull_request:
+    paths:
+    - 'plugins/**'
+    - '.github/workflows/ansible-test-sanity.yml'
+  push:
+    paths:
+    - 'plugins/**'
+    - '.github/workflows/ansible-test-sanity.yml'
+  schedule:
+  # Every day at 3:14
+  - cron:  '14 3 * * *'
+  workflow_dispatch:
+
+jobs:
+  sanity:
+    uses: data-bene/ansible-collection-gh-workflow/.github/workflows/workflow-sanity.yml@devel
```

Comment:
Since there is no matrix: how does it determine the minimal and maximal supported stable branches?

The minimum can probably be extracted from `meta/runtime.yml`'s `requires_ansible`.

The maximum should be a parameter here, and there should be a boolean flag for whether `devel` is included or not.

Reply:
I've reduced complexity but it's still very easy to add default by providing inputs parameters to the workflow.
The action currently works with a static yml file to keep versions, but it can be improved, and in fact this `meta/runtime.yml` is of interest for more automatic setup, tks for pointing to it.